libs/shared/ui/src/lib/components/button/button.tsx

```diff
@@ -10,7 +10,7 @@ const buttonVariants = cva(['inline-flex', 'items-center', 'shrink-0', 'font-med
       outline: [],
     },
     color: {
-      brand: [],
+      brand: ['disabled:text-brand-300', 'disabled:bg-brand-100', 'disabled:border-none'],
```

Comment:
Should added to line 58 instead because you can have color="brand" with variant="surface" combination

Reply:
https://github.com/Qovery/console/pull/972/commits/5b0084698dd1f61e5d859d640a96c14e8f0ea558